README.md

```diff
@@ -654,6 +654,101 @@ const App = () => {
 export default App;
 ```
 
+### Getting Setup With the Shopify Design System
+
+To start using Shopify style assets we can leverage [Polaris tokens]([https://github.com/Shopify/polaris-tokens](https://github.com/Shopify/polaris-tokens)). You can see all of the tokens [here]([https://polaris-tokens.herokuapp.com/](https://polaris-tokens.herokuapp.com/)).
+
+#### Installation
+Using  [npm](https://www.npmjs.com/):
+
+`npm install @shopify/polaris-tokens --save`
+
+Using  [yarn](https://yarnpkg.com/en/):
+
+`yarn add @shopify/polaris-tokens`
+
+#### Define Your Theme
+
+```tsx
+// In theme
+import tokens from '@shopify/polaris-tokens';
+import { createTheme } from '@shopify/restyle'
+
+const pxToNumber = (px: string) => {
+  return parseInt(px.replace('px', ''), 10);
+};
+
+export const Palette = {
```

Comment:
I think we can skip the `Palette` constant here and just point to the tokens directly in the `theme.colors` definition.